deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -330,7 +388,11 @@ def _format_tuple_param(
 async def soft_delete_jobs(
     request: Request,
     job_ids: List[uuid.UUID] = Query(None),
-    token=Depends(ProtectedAny(scope=["users:free", "users:admin"])),
+    token=Depends(
+        ProtectedAny(
+            scope=["pulse3d:user:free", "pulse3d:user:paid", "pulse3d:customer:free", "pulse3d:customer:paid"]
```

Comment:
might want to create a list of scopes for pulse3d since this gets re-used a bunch, something like

```python
PULSE3D_SCOPES = ["pulse3d:user:free", ...]
```